### PR TITLE
Update the hackathon grant fee waiver and perk card

### PR DESCRIPTION
**Pull request by rluodev**

### CODE REVIEW

File: app/views/events/promotions.html.erb [hunk 1]
```diff
@@ -198,7 +198,7 @@
           Stickers
         </h3>
         <%= image_tag "stickers/card-sticker.svg", width: 100, class: "sticker z0", style: "margin-left: -25px" %>
-        <%= image_tag "stickers/wide-sticker-light.svg", width: 150, class: "sticker wide sm-hide xs-hide ml2" %>
+        <%= image_tag "stickers/icon-sticker-dark.svg", width: 50, class: "sticker ml2 xs-hide" %>
         <%= image_tag "stickers/icon-sticker-light.svg", width: 50, class: "sticker ml2 xs-hide" %>
```
Comment (sampoder) [inline app/views/events/promotions.html.erb]: Let's split these changes off into a separate PR

File: app/views/events/promotions.html.erb [hunk 2]
```diff
@@ -198,7 +198,7 @@
           Stickers
         </h3>
         <%= image_tag "stickers/card-sticker.svg", width: 100, class: "sticker z0", style: "margin-left: -25px" %>
-        <%= image_tag "stickers/wide-sticker-light.svg", width: 150, class: "sticker wide sm-hide xs-hide ml2" %>
+        <%= image_tag "stickers/icon-sticker-dark.svg", width: 50, class: "sticker ml2 xs-hide" %>
```
Comment (cjdenio) [inline app/views/events/promotions.html.erb]:
```suggestion
        <%= image_tag "stickers/wide-sticker-light.svg", width: 150, class: "sticker wide sm-hide xs-hide ml2" %>
```
